src/stx/common/trex_rpc_cmds_common.cpp

```diff
@@ -1541,7 +1541,8 @@ TrexRpcCmdCapture::parse_cmd_fetch(const Json::Value &params, Json::Value &resul
     
     uint32_t capture_id = parse_uint32(params, "capture_id", result);
     uint32_t pkt_limit  = parse_uint32(params, "pkt_limit", result);
-    
+    uint16_t snaplen  = params.isMember("snaplen") ? parse_uint16(params, "snaplen", result) : 0;
```

Comment:
@yonghwan007 
I suggest you change this with 
```
uint16_t snaplen = parse_uint16(params, "snaplen", result, 0);
```
The `parse_uint16` function supports default values.

Reply:
I updated code as your comment.